### DISCUSSION TITLE
Do I need to include Pyinstaller's GPL in my credits?

### DISCUSSION THREAD

**Discussion by mak448a** [pyinstaller/pyinstaller#8788] [PyInstaller] [answered]
[This page](https://pyinstaller.org/en/stable/license.html) isn't too clear. Do I need to put Pyinstaller's GPL in my app's credits?

**Comment by bwoodsend** [comment, marked as answer]
No

**Reply by mak448a** [reply]
Thank you! Could you update the documentation to make it a little easier to understand?

**Reply by bwoodsend** [reply]
I'll do it anyway but I just want to reiterate that PyInstaller is a build tool, not a runtime dependency, and therefore has no more impact on the legal requirements of your application than the IDE you used to write it or the chair you happened to be sitting in when you were writing it.
